docs/articles/documentation/extending-testcafe/custom-browser-provider-plugin/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: docs
+title: Custom Browser Provider Plugin
+permalink: /documentation/extending-testcafe/custom-browser-provider-plugin/
+---
+# Custom Browser Provider Plugin
+
+TestCafe has built-in support for testing in popular browsers (see [Using Browsers](../../using-testcafe/common-concepts/using-browsers.md)).
+If you need to use a specific browser (for example, a browser from a cloud service), you can create a **custom browser provider plugin** that will allow to perform different actions on the browser windows during testing:
+to open and close it, to take screenshots of the browser window, etc.
+
+To create a browser provider plugin, use the [TestCafe browser provider generator](https://github.com/DevExpress/generator-testcafe-browser-provider).
+The generator will scaffold out the plugin, so that you only need to write a few lines of code.
+
+For more information on how to implement the plugin, see the following sections.
```

Comment:
Let's not reference section of the *current* topic with phrases like `for more information` or `for details`. They are for external references. Just say something like `to implement a plugin, you need to complete the following steps`.